### PR TITLE
release: update v2.7.2 release notes

### PR DESCRIPTION
**Pull request by cfriedt**
* add backported bugfixes
* add backported fix for CVE-2021-3966

### CODE REVIEW

**Review by stephanosio — APPROVED** [review]

**Review by MaureenHelm — APPROVED** [review]

File: doc/releases/release-notes-2.7.rst [hunk 1]
```diff
@@ -2,24 +2,132 @@
...
+* :github:`42622` - pm: pm_device structure bigger than nessecary when PM_DEVICE_RUNTIME not set
+* :github:`42825` - MQTT client disconnection (EAGAIN) on publish with big payload
+* :github:`43470` - wifi: esp_at: race condition on mutex's leading to deadlock
+* :github:`39184` - HawkBit hash mismatch
+* :github:`43490` - net: sockets: userspace accept() crashes with NULL addr/addrlen pointer
+* :github:`43853` - 2.7 branch: X86 MSI messages always get to BSP core (need a fix to be backported)
+* :github:`43807` - Test "cpp.libcxx.newlib.exception" failed on platforms which use zephyr.bin to run tests.
+* :github:`43580` - hl7800: tcp stack freezes on slow response from modem
+* :github:`42299` - spi: nRF HAL driver asserts when PM is used
+* :github:`43572` - [Backport v2.7-branch] stm32: wrong clock the LSI freq for stm32l0x mcus
```
Comment (dkalowsk) [inline doc/releases/release-notes-2.7.rst:43]: Do you want to scrub out these `[Backport]` sections?  I think this is the only one here.

Comment (cfriedt) [reply doc/releases/release-notes-2.7.rst:43]: I thought I got them already - good catch ;-)

**Comment by cfriedt** [comment]
* fixed typo
* removed last remaining `[Backport..]` prefix

**Comment by cfriedt** [comment]
* ordered closed issues numerically

File: doc/releases/release-notes-2.7.rst [hunk 2]
```diff
@@ -2,24 +2,130 @@
...
+* :github:`44337` - twister: Miss sn option to stm32cubeprogrgammer runner
+* :github:`44352` - stm32l5x boards missing the openocd runner
+* :github:`44497` - Add guide for disabling MSD on JLink OB devices and link to from smp_svr page
+* :github:`44531` - bl654_usb without mcuboot maximum image size is not limited
+* :github:`44886` - Unable to boot Zephyr on FVP_BaseR_AEMv8R
+* :github:`44902` - x86: FPU registers are not initialised for userspace (eager FPU sharing)
+* :github:`45869` - doc: update requirements
+* :github:`45870` - drivers: virt_ivshmem: Allow multiple instances of ivShMem devices
+* :github:`45871` - ci: split Bluetooth workflow
+* :github:`45872` - ci: make git credentials non-persistent
```
Comment (dkalowsk) [inline doc/releases/release-notes-2.7.rst:106]: did you want to include the CI lines here?

**Review by dkalowsk — APPROVED** [review]
Approving as the rest looks good.  If Chris wants the CI lines, let's move forward

File: doc/releases/release-notes-2.7.rst (hunk 2 above)
Comment (cfriedt) [reply doc/releases/release-notes-2.7.rst:106]: Yeah, I'm fine with that.

**Pull request merged**